### PR TITLE
Fixed bug user info not showing on static profile

### PR DESCRIPTION
**Pull request by codecopycoffee**
Fixed the bug where user info wasn't showing up on the static profile page

### CODE REVIEW

File: app/controllers/users/registrations_controller.rb [hunk 1]
```diff
@@ -16,6 +16,11 @@ def one_user
     render json: @user
   end
 
+  def static_user
```
Comment (whitecl) [inline app/controllers/users/registrations_controller.rb]: This controller action is presenting a user, not a registration. In a traditional RESTful Rails setup, this function would belong in the UsersController#show action.

Not problematic, but could be better organized in a traditional RESTful format.

Follow-up question, what is the meaning of the `static` prefix on these actions?

File: app/javascript/components/API/api.js [hunk 2]
```diff
@@ -17,3 +17,23 @@ const myRatings = id => {
 };
 
 export { myRatings };
+
+const oneStaticUser = unique_url => {
+  return fetch(`/staticuser/${unique_url}.json`).then(resp => {
+    const json = resp.json();
+
+    return json;
```
Comment (whitecl) [inline app/javascript/components/API/api.js]:
```
return resp.json();
```

is a little simpler here. There is no reason to assign a `json` variable if it is immediately returned.

File: app/javascript/components/API/api.js [hunk 3]
```diff
@@ -17,3 +17,23 @@ const myRatings = id => {
...
+  });
+};
+
+export { oneStaticUser };
```
Comment (whitecl) [inline app/javascript/components/API/api.js]: Optional - this line is unnecessary if you change line 21 to:
```
export const oneStaticUser = unique_url => {
```

File: app/controllers/users/registrations_controller.rb (hunk 1 above)
Comment (Travi5JAB) [reply app/controllers/users/registrations_controller.rb]: we added the static part to indicated that they are used for fetches on our static profile page... the profile page will have interactive parts whereas the static page will only display user info

**Pull request merged**